README.md

```diff
@@ -114,3 +114,18 @@ Any kind of contribution to ChainerRL would be highly appreciated! If you are in
 ## License
 
 [MIT License](LICENSE).
+
+## Citations
+
+If you use ChainerRL for your research, we ask that you cite the following paper:
```

Comment:
I think it's not good to *ask* for citation.

Instead we can just write `To cite ChainerRL in publications:` as https://github.com/openai/baselines

Reply:
OK. Done.